vendor/engines/split_accounts/spec/factories/split_accounts.rb

```diff
@@ -0,0 +1,45 @@
+FactoryGirl.define do
+  factory :split_account do
```

Comment:
Would it be possible to use `factory :split_acount, parent: :nufs_account` so you don't need to define quite so much here?